apps/condo/pages/user/verifyEmail.tsx

```diff
@@ -0,0 +1,181 @@
+import {
+    useVerifyUserEmailMutation,
+    useUpdateUserMutation, useStartConfirmEmailActionMutation,
+} from '@app/condo/gql'
+import { ConfirmEmailActionMessageType } from '@app/condo/schema'
+import { Row } from 'antd'
+import getConfig from 'next/config'
+import { useRouter } from 'next/router'
+import React, { useCallback, useEffect, useState } from 'react'
+
+import { getClientSideSenderInfo } from '@open-condo/miniapp-utils'
+import { useAuth } from '@open-condo/next/auth'
+
+import { AuthRequired } from '@condo/domains/common/components/containers/AuthRequired'
+import { HCaptchaProvider, useHCaptcha } from '@condo/domains/common/components/HCaptcha'
+import { Loader } from '@condo/domains/common/components/Loader'
+import { useMutationErrorHandler } from '@condo/domains/common/hooks/useMutationErrorHandler'
+import { PageComponentType } from '@condo/domains/common/types'
+import { OutdatedLinkPoster } from '@condo/domains/user/components/OutdatedLinkPoster'
+import { SuccessfulEmailVerificationPoster } from '@condo/domains/user/components/SuccessfulEmailVerificationPoster'
+
+
+const { publicRuntimeConfig: { verifyUserEmailWithMarketingConsentEnabled } } = getConfig()
+
+type ConfirmEmailPageProps = {
+    messageType?: string
+    secretCode?: string
+    token?: string
+}
+
+const VerifyEmailPage: PageComponentType<ConfirmEmailPageProps> = () => {
```

Comment:
Can it be done in `confirmEmail` page? I don't understand why we need two pages for confirm email view